pset-functions.js

```diff
@@ -33,6 +33,20 @@ console.log('Problem 1:')
 
 // Add your code above this line
 
+function blackJack(playerCardScore, dealerCardScore) {
+    const MAX = 21;
+    if(playerCardScore > MAX && dealerCardScore > MAX){
```

Comment:
@Ashbk32 great job of using this condition to immediately return 0 if either score is greater than 21